message_ix_models/model/water/data/infrastructure.py

```diff
@@ -972,25 +993,28 @@ def add_desalination(context: "Context") -> dict[str, pd.DataFrame]:
         results["output"] = out_df
 
     # putting a lower bound on desalination tecs based on hist capacities
-    df_bound = df_hist[df_hist["year"] == 2015]
+    df_bound = df_hist[df_hist["year"] == 2025]
```

Comment:
can you maybe make this based on firstmodelyear (one period before it) and the next one like this +15